src/helpers/toggle-classes-on-redirect.ts

```diff
@@ -0,0 +1,25 @@
+export const toggleClassesOnRedirect = function (isLogined: boolean = true, page: string = 'main'): void {
+  const bodyElement = document.querySelector('body');

```

Comment:
Instead of using `querySelector('body')`, consider directly accessing the body element with `document.body` to simplify the code.

```suggestion
  const bodyElement = document.body;

```